lib/redux/constants/errors.js

```diff
@@ -30,5 +30,6 @@ module.exports = {
   E_READ_FILE: 'Error reading file ',
   E_SERIAL_CONNECTION: 'Hmm, we couldn\'t detect your device',
   E_SERVER_ERR: 'Sorry, the Tidepool servers appear to be down',
-  E_UPLOAD_IN_PROGRESS: 'Sorry, an upload is already in progress'
-};
\ No newline at end of file
+  E_UPLOAD_IN_PROGRESS: 'Sorry, an upload is already in progress',
+  E_DEXCOM_UNSUPPORTED: 'Sorry, we don\'t support this Dexcom device yet'
```

Comment:
as we have been talking about language and errors does this need to be checked off do you think? Or should it say the version we don't support? Being picky but just wanted to check